com_migratetojoomla/admin/forms/main.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<form>
+	<field
+		name="mediaoptions"
+		type="list"
+		label="COM_MIGRATETOJOOMLA_SELECT_OPTION_FOR_MEDIA"
```

Comment:
Please use the 'description' attribute as well as much as possible to provide inline help